chart/templates/_helpers.tpl

```diff
@@ -84,6 +84,20 @@ app.kubernetes.io/version: {{ default $root.Chart.Version $root.Chart.AppVersion
 helm.sh/chart: {{ include "kubecf.chart" $root }}
 {{- end }}
 
+{{- /*
+==========================================================================================
+| kubecf.imagePullSecrets (list $ $component)
++-----------------------------------------------------------------------------------------
+| Add imagePullSecrets to service accounts.
+==========================================================================================
+*/ -}}
+{{- define "kubecf.imagePullSecrets" }}
+{{- $root := first . }}
```

Comment:
I wish we would properly indent templating logic everywhere.

I know the rest of this file doesn't use indentation either, so don't change anything. One day I'll rage-reindent this as a refactoring... 😄

Reply:
confirmed and indented only my section